src/modals/about-modal.directive.js

```diff
@@ -0,0 +1,135 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.modals.directive:pfAboutModal
+ *
+ * @description
+ * Directive for rendering modal windows.
+ *
+ * @param {string=} copyright Product copyright information
+ * @param {function=} onClose Function to call when modal is closed
+ * @param {string=} imgAlt The alt text for the corner grahpic
+ * @param {string=} imgSrc The source for the corner grahpic
+ * @param {boolean=} isOpen Flag indicating that the modal should be opened
+ * @param {string=} title The product title for the modal
+ *
+ * @example
+ <example module="patternfly.modals">
+   <file name="index.html">
+     <div ng-controller="ModalCtrl">
+       <button ng-click="open()" class="btn btn-default">Launch about modal</button>
+       <div pf-about-modal is-open="isOpen" on-close="onClose()"
+            title="title" copyright="copyright" img-alt="imgAlt" img-src="imgSrc">
+         <ul class="list-unstyled">
+           <li><strong>Label</strong> Version</li>
+           <li><strong>Label</strong> Version</li>
+           <li><strong>Label</strong> Version</li>
+           <li><strong>Label</strong> Version</li>
+           <li><strong>Label</strong> Version</li>
+           <li><strong>Label</strong> Version</li>
+         </ul>
+       </div>
+     </div>
+   </file>
+   <file name="script.js">
+     angular.module('patternfly.modals').controller('ModalCtrl', function ($scope) {
+       $scope.copyright = "Trademark and Copyright Information";
+       $scope.imgAlt = "Patternfly Symbol";
+       $scope.imgSrc = "img/logo-alt.svg";
+       $scope.title = "Product Title";
+       $scope.open = function () {
+         $scope.isOpen = true;
+       }
+       $scope.onClose = function() {
+         $scope.isOpen = false;
+       }
+     });
+   </file>
+ </example>
+ */
+angular.module('patternfly.modals')
+
+.directive("pfAboutModalTransclude", function ($parse) {
+  'use strict';
+  return {
+    link: function (scope, element, attrs) {
+      element.append($parse(attrs.pfAboutModalTransclude)(scope));
+    }
+  };
+})
+
+.directive('pfAboutModal', function () {
+  'use strict';
+  return {
+    restrict: 'A',
+    scope: {
+      copyright: '=?',
+      close: "&onClose",
+      imgAlt: '=?',
+      imgSrc: '=?',
+      isOpen: '=?',
+      title: '=?'
+    },
+    templateUrl: 'modals/about-modal.html',
+    transclude: true,
+    controller: ['$scope', '$modal', '$transclude', function ($scope, $modal, $transclude) {
+      // watching isOpen attribute to dispay modal when needed
+      $scope.$watch(
```

Comment:
Try to put $watch calls in the link function so that it is executed after the compile.